src/tests/ftest/cart/test_group.c

```diff
@@ -204,6 +204,25 @@ client_cb_common(const struct crt_cb_info *cb_info)
 		sem_post(&test_g.t_token_to_proceed);
 		D_ASSERT(rpc_req_output->bool_val == true);
 		break;
+	case TEST_OPC_SWIM_STATUS:
+		rpc_req_input = crt_req_get(rpc_req);
+		if (rpc_req_input == NULL)
+			return;
+		rpc_req_output = crt_reply_get(rpc_req);
+		if (rpc_req_output == NULL)
+			return;
```

Comment:
same here; change to assert as this is fatal if null

Reply:
581ebf161f1f6ab82a4ba2b619dbf33a31e5b92d